azkaban-web-server/src/main/java/azkaban/webapp/AzkabanWebServerModule.java

```diff
@@ -92,6 +93,7 @@ protected void configure() {
     bind(WebMetrics.class).to(resolveWebMetricsClass()).in(Scopes.SINGLETON);
   }
 
+  @VisibleForTesting
```

Comment:
Nit: @VisibleForTesting is not needed as we didn't change the visibility (it's still private)

Reply:
Good point. I am not using this method in test case.